docs/Command_Templates.md

```diff
@@ -27,6 +27,25 @@ Note how the `gcode:` config option always starts at the beginning of
 the line and subsequent lines in the G-Code macro never start at the
 beginning.
 
+### Add a description to your macro
+
+The help indetify the functionality a short description can be added. To
```

Comment:
Typo.

Otherwise the change is fine with me.  The `description` field will also need to be added to docs/Config_Reference.md .

-Kevin

Reply:
I hope now everything is ok